lib/v2/myfigurecollection/index.js

```diff
@@ -0,0 +1,90 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+const shortcuts = {
+    potd: 'picture/browse/potd',
+    potw: 'picture/browse/potw',
+    potm: 'picture/browse/potm',
```

Comment:
```suggestion
    potd: 'picture/browse/potd/',
    potw: 'picture/browse/potw/',
    potm: 'picture/browse/potm/',
```
Without the last `/` it is equivalent to `https://myfigurecollection.net/picture/`

Reply:
👍